src/pudl/output/pudltabl.py

```diff
@@ -129,6 +131,35 @@ def __init__(
         # Used to persist the output tables. Returns None if they don't exist.
         self._dfs = defaultdict(lambda: None)
 
+        self._register_output_methods()
+
+    def _register_output_methods(self):
+        """Load denorm assets and register methods for retrieving each asset."""
+        denorm_assets = {
+            "ferc1": load_assets_from_modules(
+                [pudl.output.denorm_ferc1], group_name="ferc1"
+            ),
+            "eia": load_assets_from_modules([pudl.output.denorm_eia], group_name="eia"),
```

Comment:
We'll probably want to load the normalized 714 and 861 assets because there are a lot of `PudlTabl` methods for these tables.

Reply:
Should we only limit the interface to the normalized FERC-714 & EIA-861 tables, which are few, or provide access to all of the tables that were previously available even though many of them are `clean` rather than `norm`?

For FERC-714 we have:
* `respondent_id_ferc714` (normalized)
* `demand_hourly_pa_ferc714` (normalized)

For EIA-861 we have:
* `balancing_authority_eia861` (normalized)
* `balancing_authority_assn_eia861` (normalized)
* `utility_assn_eia861` (normalized)
* `service_territory_eia861`(normalized)
* Everything else (including `sales_eia861` which we utilize in the state demand allocation) is still un-normalized.

Reply:
I added both of these. Right now the `clean_balancing_authority_eia861` is causing to break because it doesn't have `Resource` metadata right now

Reply:
Also, are there any assets in the 714/861 modules that we don't want to have `PudlTabl` methods? Write now it will create a method for every asset it loads, but we could make it more selective

Reply:
Oh yeah `clean_balancing_authority_eia861` is still using the default io manager. @zaneselvans is this intentional?

Reply:
Ah. Hmm. Probably `clean_balancing_authority_eia861` should be getting pickled.

Reply:
I think we want methods for all of the EIA-861 and FERC-714 tables that are written to the DB, with the exception of the `clean_balancing_authority_eia861` which is just used to create the finished, normalized `balancing_authority_eia861` table.